packages/datepicker/datepicker.scss

```diff
@@ -198,6 +199,10 @@ $bottom-margin: $component-spacing--large;
                 color: $snohvit;
                 font-weight: bold;
             }
+
+            &:focus {
+                box-shadow: 0 0 0 1px $snohvit, 0 0 0 3px $focus-color;
```

Comment:
`rem(1px)` ? Så er vi konsekvente.

Kunne også tenkt meg at vi tar i bruker animasjons-mixins når vi endrer på disse. Så kanskje vi trenger en ny verdi i den for å støtte 75ms super raske animasjoner.

Reply:
Yes, å ikke bruke rem(px) var en liten glipp.

Og enig i det andre punktet ditt. La til en på 75ms med navn "energetic"